content/en/docs/test.md

```diff
@@ -105,7 +106,7 @@ culpa qui officia deserunt mollit anim id est laborum.
     $ ls -l
     ```
   
-  - And a sub-list after some block-level content
+    - And a sub-list after some block-level content
```

Comment:
This is supposed to be a bulleted sub-item inside the parent numbered list, but this is getting rendered as a numbered sub-item instead.